backend/nlp/handler.py

```diff
@@ -1,9 +1,27 @@
 import json
+import imp
```

Comment:
Could you add some unit tests for this file? Also, can we add Python testing and linting to Travis CI?